py/kubeflow/testing/run_e2e_workflow.py

```diff
@@ -0,0 +1,230 @@
+"""Run the E2E workflow.
```

Comment:
why is the directory structured as py/kubeflow/testing/xxx? Isn't "kubeflow/testing" part unnecessary?

Reply:
We want this to be a python package that is imported as "kubeflow.testing".
The idea here is that we can add "py" to our python path and then import the code as
import kubeflow.testing...